build-system/tasks/performance/index.js

```diff
@@ -14,33 +14,75 @@
  * limitations under the License.
  */
 
-const cacheDocuments = require('./cache-documents');
-const compileScripts = require('./compile-scripts');
-const getMetrics = require('./measure-documents');
+const argv = require('minimist')(process.argv.slice(2));
 const loadConfig = require('./load-config');
-const rewriteScriptTags = require('./rewrite-script-tags');
 const runTests = require('./run-tests');
-const {installPackages} = require('../../common/utils');
+const {
+  buildMinifiedRuntime,
+  buildUnminifiedRuntime,
+  installPackages,
+} = require('../../common/utils');
+const {CONTROL, EXPERIMENT, HOST, PORT} = require('./helpers');
+const {measureDocuments, writeMetrics} = require('./measure-documents');
 const {printReport} = require('./print-report');
+const {startServer, stopServer} = require('../serve');
+
+async function setupExperimentTests() {
+  installPackages(__dirname);
+  if (!argv.nobuild) {
+    if (argv.compiled) {
+      buildMinifiedRuntime();
```

Comment:
Sanity check: Will this function eventually need to use the `--define_experiment_constant` flag being added by #27499?

Reply:
If the intent is to run against the experiment builds, yes. I think there might be other flags to think about as well? Not sure

Reply:
Not sure if that's the intent. If so, I would suggest adding it in a separate PR.